content/python/concepts/dates/terms/date/date.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'datetime.date()'
+Description: 'Returns a date object in the year-month-day format.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `datetime.date()` method returns a date object in the year-month-day format.
+
+## Syntax
+
+```pseudo
+datetime.date(YYYY, MM, DD)
+```
+
+All parameters passed to the `datetime.date()` method in the snippet above are required and must be passed in order. Otherwise, a `TypeError` is thrown.
+
+| Parameter | Description                                   |                    Range                     |
+| :-------: | --------------------------------------------- | :------------------------------------------: |
+|  `YYYY`   | The year expressed in four digits.            |                   1 - 9999                   |
+|   `MM`    | The month expressed in two digits.            |                    1 - 12                    |
```

Comment:
```suggestion
|  `YYYY`   | The year expressed in four digits.            |                   0001 - 9999                   |
|   `MM`    | The month expressed in two digits.            |                    01 - 12                    |
```

Reply:
Did we also want leading zeros? May be useful to add these for the other table on lines 27-29

Reply:
It turns out this doesn't actually affect the output! You could use leading zeroes or `date()` will automatically fill them in for you:
<img width="653" alt="Screen Shot 2022-05-27 at 3 25 52 PM" src="https://user-images.githubusercontent.com/23144845/170777593-3d2d5bb3-2eed-4446-ad54-e51a3ad1e894.png">

Reply:
Nice! I'll go ahead and resolve this